lib/overcommit/hook_context/pre_push.rb

```diff
@@ -18,17 +18,14 @@ def pushed_refs
     end
 
     def modified_files
```

Comment:
@sds ,

I want to ask your opinion on this one. If I push changes on the same file on separate branch/local_ref:

1) Should we convert the output to unique list of files.  

Or

2) Should we distinguish the file from different local_ref by appending sort of prefix. For example:

```
[‘<local_ref_1>:file_one’, ‘<local_ref_2>:file_one’]
```

Or

3) Do nothing. Let there be duplicate of file from different local_ref.

Reply:
Good question, @taufek. If I try to put myself in the shoes of a developer writing a pre-push hook, the least-surprising behavior to me would be net set of differences between the local and remote branches, and thus the unique list of files (option 1) seems reasonable to me.

If you have other ideas about why others would be better, let me know!

Reply:
Yup. Unique list of files make sense to me. Thanks.